### PR TITLE
Rename SelectionData get_data_with_length method to get_data

### PR DESCRIPTION
**Pull request by osa1**

### CODE REVIEW

**Comment by osa1** [comment]
Ping @sdroege

File: src/selection_data.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
```
Comment (EPashkin) [inline src/selection_data.rs]: 2013-2019

**Comment by EPashkin** [comment]
@osa1 Thanks
👍

File: src/selection_data.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+use SelectionData;
+use ffi;
+use glib::translate::*;
+
+impl SelectionData {
+    pub fn get_data(&self) -> Vec<u8> {
+        unsafe {
+            let mut length = mem::uninitialized();
+            let ret = FromGlibContainer::from_glib_none_num(ffi::gtk_selection_data_get_data_with_length(self.to_glib_none().0, &mut length), length as usize);
```
Comment (GuillaumeGomez) [inline src/selection_data.rs]: Will `length` be used **after** its value change from the C call or before? (I'm very curious about this!)

Comment (osa1) [reply src/selection_data.rs]: That's a good question. This code is actually auto-generated, I just copied it from the auto/ directory to here and renamed it. Perhaps it's a good idea to make it explicit by adding one more line that `length` will be used after being initialized.

Comment (GuillaumeGomez) [reply src/selection_data.rs]: What do you think about it @EPashkin @sdroege ?

Comment (sdroege) [reply src/selection_data.rs]: It will only be written to by the C function. See that it's marked as `out` via the annotations, not as `inout`.

**Comment by sdroege** [comment]
:+1:

File: src/selection_data.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply src/selection_data.rs]: It wasn't my point. The value received by `from_glib_none_mut` will be the one of `length` before its modification in the C function or after?

Comment (sdroege) [reply src/selection_data.rs]: Oh... I see. Evaluation order is defined/guaranteed, yes. Inside-out, left-to-right. In this specific case that means
1. `self.to_glib_none().0`
2. `&mut length`
3. `gtk_selection_data_get_data_with_length()`
4. `length as usize`
5. `from_glib_none_num()`
6. `ret = ...`

Comment (GuillaumeGomez) [reply src/selection_data.rs]: Thanks for the clarification!

Comment (EPashkin) [reply src/selection_data.rs]: As I know, first get_data called, then its result pushed to stack, length pushed and next `from_glib_none_num` called.

**Comment by sdroege** [comment]
@GuillaumeGomez all green apart from the examples being generally broken now :)

**Comment by GuillaumeGomez** [comment]
Oh indeed. Merging then!

**Pull request merged**